backend/services/syson-services/src/main/java/org/eclipse/syson/services/UtilService.java

```diff
@@ -761,4 +767,38 @@ private boolean isPart(Element element) {
     private boolean isAction(Element element) {
         return element instanceof ActionUsage || element instanceof ActionDefinition;
     }
+
+    /**
+     * Return an instance of the actual {@link PartDefinition} associated to the given {@link PartUsage}.
+     *
```

Comment:
Please explain here that the PartUsage parameter could be one of its substype.
That's the whole purpose of this method.

Reply:
Done